src/Tizen.NUI/src/internal/WebView/WebHttpRequestInterceptor.cs

```diff
@@ -46,17 +51,43 @@ public string Url
         {
             get
             {
-                return Interop.WebHttpRequestInterceptor.GetUrl(SwigCPtr);
+                return Interop.WebHttpRequestInterceptor.GetUrl(interceptorHandle);
+            }
+        }
+
+        /// <summary>
+        /// Gets method of intercepted http request, for example, GET, POST, etc.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Method
+        {
+            get
+            {
+                return Interop.WebHttpRequestInterceptor.GetMethod(interceptorHandle);
+            }
+        }
+
+        /// <summary>
+        /// Gets headers of intercepted http request, for example, "schema: https".
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public PropertyMap Headers
```

Comment:
The result data is a set of string key-value pair?

Reply:
Yes. I updated comments.

Reply:
How about use Dictionary<string,string> instead PropertyMap
ProperyMap is actually not map data structure.

Reply:
I agree with him. I think Dictionary is more intuitive in this case.

Reply:
Done.